ts/src/tokocrypto.ts

```diff
@@ -1607,7 +1608,10 @@ export default class tokocrypto extends Exchange {
                     request['quoteOrderQty'] = this.decimalToPrecision (quoteOrderQty, TRUNCATE, precision, this.precisionMode);
                     params = this.omit (params, [ 'quoteOrderQty', 'cost' ]);
                 } else if (price !== undefined) {
-                    request['quoteOrderQty'] = this.decimalToPrecision (amount * price, TRUNCATE, precision, this.precisionMode);
+                    const amountString = this.numberToString (amount);
```

Comment:
@samgermain not directly related to this, but regular market orders are not working, can you check it?
```
n tokocrypto createOrder "LTC/USDT" market buy 0.1
Debugger attached.
2023-06-30T10:12:18.518Z
Node.js: v18.14.0
CCXT v3.0.72
tokocrypto.createOrder (LTC/USDT, market, buy, 0.1)
InvalidOrder tokocrypto {"code":3203,"msg":"Incorrect Order Quantity","timestamp":1688119942051}
---------------------------------------------------
[InvalidOrder] tokocrypto {"code":3203,"msg":"Incorrect Order Quantity","timestamp":1688119942051}

    at throwExactlyMatchedException  Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:2791  
    at handleErrors                  Users/cjg/Git/ccxt10/ccxt/js/src/tokocrypto.js:2419     
    at                               Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:748   
    at processTicksAndRejections     node:internal/process/task_queues:95                    
    at fetch2                        Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:2433  
    at request                       Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:2436  
    at createOrder                   Users/cjg/Git/ccxt10/ccxt/js/src/tokocrypto.js:1650     
    at async run                     Users/cjg/Git/ccxt10/ccxt/examples/js/cli.js:295        

tokocrypto {"code":3203,"msg":"Incorrect Order Quantity","timestamp":1688119942051}
```

Reply:
> @samgermain not directly related to this, but regular market orders are not working, can you check it?
> 
> ```
> n tokocrypto createOrder "LTC/USDT" market buy 0.1
> Debugger attached.
> 2023-06-30T10:12:18.518Z
> Node.js: v18.14.0
> CCXT v3.0.72
> tokocrypto.createOrder (LTC/USDT, market, buy, 0.1)
> InvalidOrder tokocrypto {"code":3203,"msg":"Incorrect Order Quantity","timestamp":1688119942051}
> ---------------------------------------------------
> [InvalidOrder] tokocrypto {"code":3203,"msg":"Incorrect Order Quantity","timestamp":1688119942051}
> 
>     at throwExactlyMatchedException  Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:2791  
>     at handleErrors                  Users/cjg/Git/ccxt10/ccxt/js/src/tokocrypto.js:2419     
>     at                               Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:748   
>     at processTicksAndRejections     node:internal/process/task_queues:95                    
>     at fetch2                        Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:2433  
>     at request                       Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:2436  
>     at createOrder                   Users/cjg/Git/ccxt10/ccxt/js/src/tokocrypto.js:1650     
>     at async run                     Users/cjg/Git/ccxt10/ccxt/examples/js/cli.js:295        
> 
> tokocrypto {"code":3203,"msg":"Incorrect Order Quantity","timestamp":1688119942051}
> ```

it looks like you either need to provide the price or give a larger order quantity

Reply:
@samgermain maybe for market-buy it expects the cost? if so we have to fix it with the createMarketBuyRequriesPrice thing